src/main/java/com/capstone/producer/TfrHandler.java

```diff
@@ -0,0 +1,179 @@
+        Matcher matcher = pattern.matcher(notam);
+        if (matcher.find()) {
+            return matcher.group();
+        }
+        return null;
+    }
+
+    private static boolean addToHashMap(String notamNumber, String notam){
+        String[] arryBuilder = null;
+        //check if notam currently exists in map
+        if (receivedNotams.containsKey(notamNumber)){
+            arryBuilder = receivedNotams.get(notamNumber);
+        }
+
+        Pattern pattern = Pattern.compile("(PART\s)(\\d+)(\sOF\s)(\\d+)");
+        Matcher matcher = pattern.matcher(notam);
+        
+
+        // if matcher has a string, we have a multipart notam.
+        // if not, it is a single part notam
+        if(matcher.find()){
+            LOGGER.debug("Processing TFR NOTAM PART {} of {}", matcher.group(2), matcher.group(4));
+            if(arryBuilder == null){
+                arryBuilder = new String[Integer.parseInt(matcher.group(4))];
+            }
+            arryBuilder[Integer.parseInt(matcher.group(2)) - 1] = notam;
+            receivedNotams.put(notamNumber, arryBuilder);
+            //test if array is filled out
+            for(int i = 0; i < arryBuilder.length; i++) {
+                if(arryBuilder[i] == null)
+                    return false;
+            }
+            return true;
+        }
+        // single message NOTAM
+        arryBuilder = new String[1];
+        arryBuilder[0] = notam;
+        receivedNotams.put(notamNumber, arryBuilder );
+        return true;
+    }
+
+    private static String combineNotamMessages(String notamNumber) {
+        String[] allMessages = receivedNotams.get(notamNumber);
+        String fullMessage = "";
+        Pattern pattern = Pattern.compile("(FDC\\s*\\d/\\d{4}\\s*.*..TEMPORARY\\s*(FLIGHT)?\\s*(RESTRICTIONS.)?)(.*)(\\b\\d{10}-\\d{10})");
+        String stringToParse = "";
+        for(int i = 0; i < allMessages.length; i++) {
+            LOGGER.info("AllMessage of message: {}", allMessages[i]);
+            stringToParse = allMessages[i].replaceAll(System.getProperty("line.separator"), " ");
+            LOGGER.info("String to parse: {}", stringToParse);
+            Matcher matcher = pattern.matcher(stringToParse);
+            if(matcher.find()) {
+                fullMessage += matcher.group(4);
+            }
+        }
+        combinedNotams.put(notamNumber, fullMessage);
+        return fullMessage;
+    }
+
+    private static boolean pointRadiusParse(String notamNumber, String message) throws InterruptedException{
+        Pattern pattern = Pattern.compile("(WI\\s*AN\\s*AREA\\s*DEFINED\\s*AS\\s*(\\d*.?\\d*)NM\\s*RADIUS\\s*OF\\s*(\\d+[NS]\\d+[EW])(.*?)EFFECTIVE\\s*(\\d{10})\\s*UTC.*?UNTIL\\s*(\\d{10})?)");
+        Matcher matcher = pattern.matcher(message);
+        boolean successfulMatching = false;
+        while(matcher.find()){ //Allows for multiple finds.
+            LOGGER.info("String matched Radius test: {}", matcher.group(0));
```

Comment:
I think most of these log info should probably be `debug`. They are concern with some inner working of the code rather than an overview of what is going on in the application. They will most likely only ever be relevant to dev as they are debugging.

Reply:
Changed to debug